modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/cep/integration/common/utils/CEPIntegrationTest.java

```diff
@@ -89,13 +97,13 @@ protected void gracefullyRestartServer() throws Exception {
 
     /**
      * @param testCaseFolderName Name of the folder created under /artifacts/CEP for the particular test case.
-     * @param configFileName Name of the XML config-file created under above folder.
+     * @param configFileName     Name of the XML config-file created under above folder.
      * @return The above XML-configuration, as a string
      * @throws Exception
      */
     protected String getXMLArtifactConfiguration(String testCaseFolderName, String configFileName)
             throws Exception {
-        String relativeFilePath = getTestArtifactLocation() + "/artifacts/CEP/"+testCaseFolderName+"/"+configFileName;
+        String relativeFilePath = getTestArtifactLocation() + "/artifacts/CEP/" + testCaseFolderName + "/" + configFileName;
```

Comment:
And also as a practice don't put string literals in the code like this. Use the constants class to put these all literals there and reference them from here.